pkg/cli/admin/inspect/inspect.go

```diff
@@ -10,6 +10,8 @@ import (
 	"strings"
 	"time"
 
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
Nit: I like those imports nicely grouped into the following groups:
1. golang imports
2. 3-rd party
3. k8s imports
4. openshift imports